### DISCUSSION TITLE
Add Gnosis Chain support

### DISCUSSION THREAD

**Discussion by vporton** [wevm/wagmi#1544] [Ideas]
Add Gnosis Chain support:

I have already created a feature branch implementing it:
https://github.com/vporton/wagmi/tree/feature-gnosis

Please allow to create a pull request.

**Comment by jxom** [comment]
Gnosis is now in `wagmi@0.9.5` thanks to [your PR](https://github.com/wagmi-dev/references/pull/26). 😊
